# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1337888339 14400
#      Thu May 24 15:38:59 2012 -0400
# Node ID 22244a235fd09f9d17898508666bd2491a6aecf8
# Parent  57569a35765c076c62594a682e5a987ec25b3b87
# Parent  847ed7f603cfc48aee0546ec1a7007184e797772
maint: periodic merge of stable to default

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -60,8 +60,9 @@ 72aebe6196414e38ef802469ff6e238f914f04c9
 64d9f33313cc8c691974bcd123357e24bccbabdc rc-3-6-0-1
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3.6.0
 95c43fc8dbe1a07a46fefb3372df5b2309d874fd rc-3-6-1-0
 0000000000000000000000000000000000000000 release-3-2-4
 e320928eeb3aa2370b792e83dafc3e0ddecdc871 release-3-2-4
 ba4d6343524b406b0d15aee34579f80783581c54 release-3-6-1
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3-6-0
 f947d2922febf12dcd1fb6e21b356756ecb54e55 rc-3-6-2-0
+4460c4fb20e6a5d3b1972fa737d4e00eb921545a rc-3-6-2-2
